### PR TITLE
Enable plotting from tiled and launching a latent space extraction through an autoencoder

### PR DESCRIPTION
**Pull request by taxe10**
This PR addresses some minor bugs in the current version of latent space explorer:
- Enables plotting tiled data when selecting data points from the latent space plot
- It corrects the orientation of the data, which were upside down
- Enables running the application outside of docker through the definition of env variables
- Replaced env file for env.example file and updated gitignore file
- Rearranged plots to be side-by-side
- Added an optional autoencoder inference step to be run prior dimension reduction
- Added job dropdown with previously ran jobs
- Added conda flows as an alternative to podman jobs

### CODE REVIEW

**Comment by runboj** [comment]
Hi Tanny, what is "CONTENT_API_URL" used for? @taxe10

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+FLOW_NAME="Parent flow/launch_parent_flow"
+TIMEZONE="US/Pacific"
+PREFECT_TAGS='["latent-space-explorer"]'
+
+CONTENT_API_URL="http://localhost:8000/api/v0/models"
+
+TILED_API_KEY=<api key>
+
+DATA_DIR=/path/to/data
+OUTPUT_DIR=/path/to/output  # optional - if not provided, will default to data
```
Comment (runboj) [inline .env.example]: Do we need output_dir?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,16 @@
+USER=admin
+DEFAULT_ALGORITHM_DESCRIPTION=/path/to/PCA_v1.0.0.json
+
+PREFECT_API_URL=http://localhost:4200/api
+FLOW_NAME="Parent flow/launch_parent_flow"
+TIMEZONE="US/Pacific"
+PREFECT_TAGS='["latent-space-explorer"]'
+
+CONTENT_API_URL="http://localhost:8000/api/v0/models"
```
Comment (runboj) [inline .env.example]: This is the content registry api

**Comment by taxe10** [comment]
> Hi Tanny, what is "CONTENT_API_URL" used for? @taxe10

The CONTENT_API_URL is used to set up the url to the content registry api, which could be "localhost:8000" outside docker or "content_registry:8000" within docker, or any other custom setups

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+TILED_API_KEY=<api key>
+
+DATA_DIR=/path/to/data
```
Comment (runboj) [inline .env.example]: For this DATA_DIR, can you give me an example way to format it? Would "/app/work/data" work when running in Docker on my mac?

**Review by runboj — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (taxe10) [reply .env.example]: No, but I decided to restructure the variables to `READ_DIR` and `WRITE_DIR`, where each define a directory path to read files from and the other defines the directory path where to store results

File: .env.example (hunk 3 above)
Comment (taxe10) [reply .env.example]: In my machine, the READ_DIR is set to `/Users/tanny/Documents/Data` as I have some few datasets stored there, and the WRITE_DIR is set to `/Users/tanny/Documents/Results`

**Pull request merged**